### ISSUE TITLE
javascript API?

### ISSUE THREAD

**Issue by jaredmcdonald** [adafruit/adafruit-io-node#12]
is there still a public javascript API, as hinted at in [this tutorial](https://learn.adafruit.com/adafruit-io-basics-digital-output/raspberry-pi#node-dot-js-script)? if so, it seems to have changed since that writing--some documentation on the JS API would be greatly appreciated 😄

thanks!

**Comment by siliconchris1973** [comment]
Yes me too. I would love to include adafruit-io-node in my node.js application but have no idea where to start.
